compiler/hash-semantics/src/passes/discovery/defs.rs

```diff
@@ -485,9 +500,10 @@ impl<'tc> DiscoveryPass<'tc> {
                 buf.push((
                     node.id(),
                     Decl {
-                        name: sym(binding.name.ident),
-                        // is_mutable: binding.mutability.as_ref().map(|m| *m.body())
```

Comment:
Maybe a leave a comment that mutability should be added here in `@@Future`?

Reply:
I don't think that's necessary tbh, we have to figure out mutability/memory as a whole anyway, which will probably require additions in lots of places..